src/index.html

```diff
@@ -11,6 +11,50 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
```

Comment:
Inputs should have some empty space from both sides:
![image](https://user-images.githubusercontent.com/86923089/209427549-4045141e-2300-4b52-bf8e-3b1070e5e721.png)